src/components/common/profile/ProfileCard.tsx

```diff
@@ -0,0 +1,188 @@
+import React from 'react';
+
+import useMediaQuery from '@/hooks/useMediaQuery';
+
+import ImageBox from '../ImageBox';
+
+interface ProfileCardProps {
+  name: string;
+  bio?: string;
+  imageUrl?: string;
+  isEditing: boolean;
+  onEdit: () => void;
+  onCancel: () => void;
+  onSave: () => void;
+  onImageChange: (newImageUrl: string) => void;
+  onNameChange: (newName: string) => void;
+  onBioChange: (newBio: string) => void;
+  canEdit: boolean;
+}
+
+export default function ProfileCard({
+  name,
+  bio,
+  imageUrl,
+  isEditing,
+  onEdit,
+  onCancel,
+  onSave,
+  onImageChange,
+  onNameChange,
+  onBioChange,
+  canEdit
+}: ProfileCardProps) {
+  const isDesktop = useMediaQuery('(min-width: 1024px)');
+
+  const displayBio = bio || '한 줄 소개가 없습니다.';
+  const displayImageUrl = imageUrl || '/icons/profile-default.svg';
+
+  return (
+    <div className={`bg-white rounded-lg p-4 shadow-md ${isDesktop ? 'w-96' : 'w-full'}`}>
+      {isDesktop ? (
+        <div className="flex flex-col items-center">
+          <div className="relative flex h-32 w-32 items-center justify-center rounded-full bg-gray-200">
+            <ImageBox
+              src={displayImageUrl}
+              alt="Profile Image"
+              width={32}
+              height={32}
+              className="h-full w-full rounded-full"
+            />
+            {canEdit && isEditing && (
+              <button
+                type="button"
+                // eslint-disable-next-line max-len
+                className="absolute bottom-0 right-0 flex h-8 w-8 items-center justify-center rounded-full border-solid border-gray-02 bg-white-01"
+              >
+                <ImageBox src="/icons/camera.svg" alt="Camera Icon" width={8} height={8} className="rounded-full" />
```

Comment:
isDesktop이 true이거나 false일 때의 변화가 버튼과 ImageBox의 존재 여부인 거 같습니다.

```suggestion
              { isDesktop && (<button
                type="button"
                // eslint-disable-next-line max-len
                className="absolute bottom-0 right-0 flex h-8 w-8 items-center justify-center rounded-full border-solid border-gray-02 bg-white-01"
              >
                <ImageBox src="/icons/camera.svg" alt="Camera Icon" width={8} height={8} className="rounded-full" />
                )}
```

그외 css 변화는 `md:`, `lg:`와 같은 가상 선택자를 사용해 보는 건 어떤가요?